templates/vue/src/utils/test.js

```diff
@@ -1,7 +1,35 @@
 import { render as r } from "@testing-library/vue"
 import { store } from "@/store"
-import { parse } from "./dataset"
+import { routes } from "@/router"
+import { parse, makeMockProgress } from "./dataset"
+import Helpers from "./Helpers"
 
+export function render(
+  component,
+  { fixture = null, ...options } = {},
+  callback = Helpers.noop
```

Comment:
Can we not just do this to keep it more simple:
```suggestion
  callback = ()=>{}
```

Reply:
Sure.